src/test/java/com/digirati/taxonomy/manager/lookup/persistence/SkosTranslatorTest.java

```diff
@@ -0,0 +1,203 @@
+
+    private static final RdfModel conceptSchemesRdfModel = createConceptSchemesModel();
+
+    private static final RdfModel conceptsRdfModel = createConceptsModel();
+
+    private SkosTranslator underTest = new SkosTranslator();
+
+    @Test
+    void shouldExtractConceptSchemesFromRdf() {
+        // Given
+        InputStream testConceptSchemesInput = load("test-concept-schemes-input.jsonld");
+
+        // When
+        RdfModel actual = underTest.translate(testConceptSchemesInput, null, SkosFileType.JSON_LD);
+
+        // Then
+        assertEquals(conceptSchemesRdfModel, actual);
+    }
+
+    @Test
+    void shouldSerialiseConceptSchemesToRdf() throws IOException {
+        // When
+        Model actualModel = underTest.translate(conceptSchemesRdfModel);
+        StringWriter actualWriter = new StringWriter();
+        actualModel.write(actualWriter, SkosFileType.JSON_LD.getFileTypeName());
+        String actual = actualWriter.toString().trim();
+
+        // Then
+        InputStream expectedStream = load("test-concept-schemes-output.jsonld");
+        String expected = IOUtils.toString(expectedStream, StandardCharsets.UTF_8).trim();
+
+        assertEquals(expected, actual);
+    }
+
+    @Test
+    void shouldExtractConceptsFromRdf() {
+        // Given
+        InputStream testConceptsInput = load("test-concepts-input.jsonld");
+
+        // When
+        RdfModel actual = underTest.translate(testConceptsInput, null, SkosFileType.JSON_LD);
+
+        // Then
+        assertEquals(conceptsRdfModel, actual);
+    }
+
+    @Test
+    void shouldSerialiseConceptsToRdf() throws IOException {
+        // When
+        Model actualModel = underTest.translate(conceptsRdfModel);
+        StringWriter actualWriter = new StringWriter();
+        actualModel.write(actualWriter, SkosFileType.JSON_LD.getFileTypeName());
+        String actual = actualWriter.toString().trim();
+
+        // Then
+        InputStream expectedStream = load("test-concepts-output.jsonld");
+        String expected = IOUtils.toString(expectedStream, StandardCharsets.UTF_8).trim();
+
+        assertEquals(expected, actual);
+    }
+
+    private static RdfModel createConceptSchemesModel() {
+        ConceptSchemeModel conceptSchemeModel =
+                new ConceptSchemeModel().setIri("http://example.com/");
+
+        ConceptModel one =
+                new ConceptModel()
+                        .setIri("http://example.com/concept#1")
+                        .setPreferredLabel(
+                                "[{\"language\":\"fr\",\"value\":\"un\"},{\"language\":\"en\",\"value\":\"one\"}]");
+        ConceptModel two =
+                new ConceptModel()
+                        .setIri("http://example.com/concept#2")
+                        .setPreferredLabel(
+                                "[{\"language\":\"fr\",\"value\":\"deux\"},{\"language\":\"en\",\"value\":\"two\"}]");
+
+        ConceptSemanticRelationModel schemeHasTopConceptOne =
```

Comment:
My plan here was to get it working and then tidy it up after - this was put in place when I was first playing around with how it would work and I've not got round to doing a general tidy up yet